tests/components/media_player/test_init.py

```diff
@@ -325,7 +325,9 @@ async def test_get_async_get_browse_image_quoting(
         "homeassistant.components.media_player.MediaPlayerEntity."
         "async_get_browse_image",
     ) as mock_browse_image:
-        media_content_id = "a/b c/d+e%2Fg{}"
```

Comment:
`/` is never supposed to be encoded in the URL as its a separator.

We could decode them ourselves but the change was done for security reasons so I didn't do that and we can assess if someone comes up with a valid use case.